Build the kernel module as openafs.ko instead of libafs.ko.  This should
really be done upstream as well, but it poses backwards compatibility
problems and it's always been called libafs on other platforms, so it's
controversial.

--- openafs.orig/src/config/linux-version
+++ openafs/src/config/linux-version
@@ -38,7 +38,7 @@
 CAN_BUILD=""
 
 for VERS in $LINUX_VERS ; do
-	dir=$LINUX_SRCDIR$VERS
+	dir=$LINUX_SRCDIR
 	if [ ! -d $dir ] ; then
 	    dir=$LINUX_SRCDIR
 	    if [ ! -d $dir ] ; then
@@ -47,7 +47,7 @@
 		continue
       	    fi
 	fi
-	header=$LINUX_SRCDIR$VERS/include/linux/version.h
+	header=$LINUX_SRCDIR/include/linux/version.h
 	if [ ! -f $header ] ; then
 	    header=$LINUX_SRCDIR/include/linux/version.h
 	    if [ ! -f $header ] ; then
--- openafs.orig/src/libafs/MakefileProto.LINUX.in
+++ openafs/src/libafs/MakefileProto.LINUX.in
@@ -222,8 +222,8 @@
 
 # Below this line are targets when in the COMMON directory:
 # For Linux there is no kernel NFS server.
-LIBAFS = libafs-${CLIENT}.${LINUX_MODULE_EXT}
-LIBAFS_MP = libafs-${CLIENT}.mp.${LINUX_MODULE_EXT}
+LIBAFS = openafs.${LINUX_MODULE_EXT}
+LIBAFS_MP = openafs.${LINUX_MODULE_EXT}
 LIBAFS_EP = libafs-${CLIENT}.ep.${LINUX_MODULE_EXT}
 LIBAFS_BM = libafs-${CLIENT}.bm.${LINUX_MODULE_EXT}
 
@@ -232,10 +232,8 @@
 INST_LIBAFS_EP = ${DESTDIR}${afskerneldir}/${LIBAFS_EP}
 INST_LIBAFS_BM = ${DESTDIR}${afskerneldir}/${LIBAFS_BM}
 
-DEST_LIBAFS = ${DEST}/root.client/usr/vice/etc/modload/${LIBAFS}
-DEST_LIBAFS_MP = ${DEST}/root.client/usr/vice/etc/modload/${LIBAFS_MP}
-DEST_LIBAFS_EP = ${DEST}/root.client/usr/vice/etc/modload/${LIBAFS_EP}
-DEST_LIBAFS_BM = ${DEST}/root.client/usr/vice/etc/modload/${LIBAFS_BM}
+DEST_LIBAFS = ${DEST}/root.client/usr/vice/etc/modload/openafs.o
+DEST_LIBAFS_MP = ${DEST}/root.client/usr/vice/etc/modload/openafs.mp.o
 
 
 libafs:	$(LIBAFS) 
@@ -251,11 +249,11 @@
 	echo BM Build Complete
 
 <linux26 linux_26 umlinux26>
-${LIBAFS} ${LIBAFS_MP} ${LIBAFS_EP} ${LIBAFS_BM}: libafs.ko
-	cp libafs.ko $@
+${LIBAFS} ${LIBAFS_MP} ${LIBAFS_EP} ${LIBAFS_BM}: openafs.ko
+	cp openafs.ko $@
 
 .FORCE:
-libafs.ko: .FORCE
+openafs.ko: .FORCE
 	env EXTRA_CFLAGS="${EXTRA_CFLAGS}" @TOP_SRCDIR@/libafs/make_kbuild_makefile.pl ${KDIR} $@ @TOP_OBJDIR@/src/config/Makefile.config Makefile.afs Makefile.common
 	env EXTRA_CFLAGS="${EXTRA_CFLAGS}" $(MAKE) -C ${LINUX_KERNEL_PATH} M=@TOP_OBJDIR@/src/libafs/${KDIR} modules
         
